spec/modules/helpers.rb

```diff
@@ -71,4 +81,36 @@ def delete_category(category_name, category_description)
     expect(page).to have_text("Category was successfully destroyed.")
     expect(page).not_to have_selector("#category_#{category_id}")
   end
+end
+
+module DiscussionHelpers
+  include SharedMatchers
+  include DiscussionMatchers
+
+  def create_discussion(discussion_title, discussion_body)
+    visit category_path(Category.create(name: DEFAULT_CATEGORY_NAME, position: 1, description: CATEGORY_DESCRIPTION))
```

Comment:
What do you think about passing in the category as a parameter?  This prevents manipulation of the category as part of the test setup.